### ISSUE TITLE
Refining of functional breakpoints needed

### ISSUE THREAD

**Issue by neuromusic** [chanzuckerberg/napari-hub#162]
*from June 2020 pre-launch bug bash*

**Created by**: Lia

## Description 

Since the breakpoints that exist exclusively for the home page are based on the width that things are rendered, they need a little refining since the rendering of some elements in HTML is not exactly the same as the design (i.e. spacing between bullet disc and bullet text, etc). This is fine, but slightly affects the point at which the breakpoint itself should occur. 

Work with Lia to refine. Dependent on #66 being completed first

## Info

**Bug source**: <875px

**Browser**: *unknown*

**PR**: *none*

## Attachments

https://dl.airtable.com/.attachments/e9db5ba7de6fd2a90f43b1d42583e77e/29a11dd5/ScreenShot2021-06-16at3.31.43PM.png

**Comment by codemonkey800** [comment]
Discussed with @liaprins-czi and agreed that this can be marked lower priority because the issues occur on uncommon screen widths. It's something we can work on in the future though when we have bandwidth 😄
